core/src/test/kotlin/fr/sncf/osrd/stdcm/preprocessing/BlockAvailabilityTests.kt

```diff
@@ -54,7 +54,7 @@ class BlockAvailabilityTests {
             "zone.[det.center.1:INCREASING, det.center.2:DECREASING]",
             "zone.[det.center.2:INCREASING, det.center.3:DECREASING]",
             "zone.[det.b1.nf:DECREASING, det.b2.nf:DECREASING, det.center.3:INCREASING]",
-            "zone.[det.b1.nf:INCREASING, bf.b1:DECREASING]"
+            "zone.[bf.b1:DECREASING, det.b1.nf:INCREASING]"
```

Comment:
do you know why this change is required?

Reply:
The sorting for naming is changed in this commit: https://github.com/osrd-project/osrd/pull/6921/commits/a5da254c30e3038dd14b0c361d7ff5d340593edd#diff-5d11fb62d051b79ae994acaff5b58f193131909011f76ad3547e4c777e4dfc85R389
This change was done to stabilize names and ease comparison.